source/crud/tests/unified/aggregate.yml

```diff
@@ -0,0 +1,79 @@
+description: "aggregate"
```

Comment:
This and `find.yml` ensure that cursors work even with multiple batches (test required by scope).